platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -9,20 +9,26 @@ Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
 ## Required accessibility checklist for staging review
```

Comment:
In the second paragraph of the Accessibility staging review preparation, there is this line:

> Accessibility specialists conduct a [more thorough pre-launch audit](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-request-prelaunch-review.md) when an application is relatively code-stable in production.)

We no longer conduct this Pre-launch 508 Review, and it was replaced with the Staging Review (code stable in staging) and Full A11y & 508 Office Audit (after launch).

I believe the Pre-launch 508 Review was more effective for catching accessibility issues, reducing the number found post-launch. The 2-day turnaround requirement of the staging review led to removing this.

Reply:
Updated the URL and changed this to post-launch language